bin/iuf-skopeo-sync

```diff
@@ -82,10 +81,12 @@ process(){
             image_dir="$(dirname "$relative_manifest_path")"
             image_name="$(cut -d":" -f 1 <<< "$image_dir")"
             image_version="$(cut -d":" -f 2 <<< "$image_dir")"
-            echo "INFO recording results for ${image_name}:${image_version}"
+            echo "INFO Adding entry in 'records.yaml' file for image: ${image_name}:${image_version}"
```

Comment:
again, debug for "i'm going to", info for success, error for failure

Reply:
Done